src/py-opentimelineio/opentimelineio-bindings/otio_imath.cpp

```diff
@@ -101,7 +101,7 @@ static void define_imath_2d(py::module m) {
             });
 
     py::class_<IMATH_NAMESPACE::Box2d>(m, "Box2d", py::module_local())
-        .def(py::init<>())
+        .def(py::init<>([]() { return IMATH_NAMESPACE::Box2d(IMATH_NAMESPACE::V2d(0.0, 0.0)); }))
         .def(py::init<IMATH_NAMESPACE::V2d>())
```

Comment:
@reinecke In the meeting today we were wondering if this also needs to be initialized:
```
.def(py::init<IMATH_NAMESPACE::V2d>())
```
I believe this is OK as is, since this constructs a `Box2d` from a `V2d`. Since the `V2d` is now initialized to zero with this change, the `Box2d` will also be initialized to zero:
```
v = otio.schema.V2d() # initialized to zero
box = otio.schema.Box2d(v) # also initialized to zero
```